kedro-org/kedro-website
# Feature/testimonials

## Description

Resolves #10.

## Development notes

The testimonials section is a carousel. I've used Framer Motion for the animation and have based it on [this example](https://codesandbox.io/s/framer-motion-image-gallery-pqvx3?file=/src/Example.tsx).

Since this is static content, I haven't added any tests.

I also added a `Makefile` with a `sign-off` command so we don't have to deal with the DCO check anymore.

## QA notes

**Please note** that the implementation doesn't exactly match the layout in the Figma file. We decided to place the control dots underneath the carousel slides on all screen sizes, not just on mobile, as it shows in Figma.

Please refer to [this prototype](https://www.figma.com/proto/3OU7t2Pk3t1pO5eaVi3z0M/Kedro-Prototypes?node-id=421%3A204&scaling=min-zoom&page-id=398%3A70&starting-point-node-id=398%3A182) to see what the design should look like on medium and large screen sizes and to see an approximation of the animation.

Another note: the animation doesn't exactly match the prototype. It follows it closely and makes some subtle tweaks that feel better when swiping to change the slides.

Lastly, I don't have the correct user photos or company logos at the time of writing. I don't believe that's enough to prevent me from opening this PR, so when I get the assets, I'll update the site and this text accordingly.

## Checklist
